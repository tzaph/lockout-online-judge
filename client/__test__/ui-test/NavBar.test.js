import React from "react";
import renderer from "react-test-renderer";
import NavBar from "../../src/components/NavBar";
import { BrowserRouter as Router } from 'react-router-dom';

describe("<NavBar />", () => {
  it("Renders correctly", () => {
    const tree = renderer.create(<Router><NavBar /></Router>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
