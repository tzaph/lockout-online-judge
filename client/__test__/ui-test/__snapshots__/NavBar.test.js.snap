// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar /> Renders correctly 1`] = `
<nav
  className="nav"
>
  <a
    className="site-title"
    href="/"
    onClick={[Function]}
  >
    Lockout OJ
  </a>
  <ul>
    <li
      className=""
    >
      <a
        href="/problem-recommendation"
        onClick={[Function]}
      >
        Problem Recommendation
      </a>
    </li>
    <li
      className=""
    >
      <a
        href="/room-list"
        onClick={[Function]}
      >
        Join Room
      </a>
    </li>
  </ul>
</nav>
`;
